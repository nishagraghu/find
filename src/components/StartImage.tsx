import React from 'react';
import { View, Image } from 'react-native';
import myImage from '../../assets/login.jpg';
const StartImage : React.FC  = () => {
  return (
    <View>
      <Image
        source={myImage}
        style={{ width: 400, height: 400 }}
      />
    </View>
  );
};

export default StartImage;
