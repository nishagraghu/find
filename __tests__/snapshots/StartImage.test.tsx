import React from 'react';
import { render } from '@testing-library/react-native';

import StartImage from '../../src/components/StartImage'


describe('StartImage component', () => {
  test('renders correctly', () => {
    const { toJSON } = render(<StartImage />);
    expect(toJSON()).toMatchSnapshot();
  });
});
