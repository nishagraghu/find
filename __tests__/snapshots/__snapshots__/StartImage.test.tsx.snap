// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartImage component renders correctly 1`] = `
<View>
  <Image
    source={1}
    style={
      {
        "height": 400,
        "width": 400,
      }
    }
  />
</View>
`;
